algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -441,23 +441,88 @@ case class CMSZero[K](override val params: CMSParams[K]) extends CMS[K](params)
  */
 case class CMSItem[K](item: K, override val totalCount: Long, override val params: CMSParams[K]) extends CMS[K](params) {
 
-  override def +(x: K, count: Long): CMS[K] = CMSInstance[K](params) + item + (x, count)
+  override def +(x: K, count: Long): CMS[K] = CMSInstance[K](params) + (item, totalCount) + (x, count)
```

Comment:
By all means, actually I think "override def +(x: K, count: Long): CMS[K] = CMSInstance[K](params) + (item, totalCount) + (x, count)" also is not correct.
Should be using SparseCMS here "override def +(x: K, count: Long): CMS[K] = SparseCMS[K](params) + (item, totalCount)  + (x, count)"
I will change this.